content/reference/sql/version-control/dolt-system-tables.md

```diff
@@ -756,6 +757,141 @@ WHERE  to_commit='pu60cdppae7rumf1lm06j5ngkijp7i8f';
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | rankings_post_season         | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | rankings_regular_season      | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | season_totals_allstar        | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | season_totals_post_season    | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | season_totals_regular_season | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | career_totals_allstar        | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | career_totals_post_season    | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | career_totals_regular_season | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | rankings_post_season         | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | rankings_regular_season      | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | season_totals_allstar        | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | season_totals_post_season    | gp          |
+| chcg5skg8gb1be475bmab5uuucsb8ps7 | season_totals_regular_season | gp          |
+| o7p3tki7kc7v8s40s0dtn1mf6rksh1qt | career_totals_allstar        | gp          |
+| o7p3tki7kc7v8s40s0dtn1mf6rksh1qt | career_totals_post_season    | gp          |
+| o7p3tki7kc7v8s40s0dtn1mf6rksh1qt | career_totals_regular_season | gp          |
+| o7p3tki7kc7v8s40s0dtn1mf6rksh1qt | rankings_post_season         | gp          |
+| o7p3tki7kc7v8s40s0dtn1mf6rksh1qt | rankings_regular_season      | gp          |
+| o7p3tki7kc7v8s40s0dtn1mf6rksh1qt | season_totals_allstar        | gp          |
+| o7p3tki7kc7v8s40s0dtn1mf6rksh1qt | season_totals_post_season    | gp          |
+| o7p3tki7kc7v8s40s0dtn1mf6rksh1qt | season_totals_regular_season | gp          |
+| ii54ianm3v79gn67r90uj9usm2fsm1v2 | career_totals_post_season    | gp          |
+| ii54ianm3v79gn67r90uj9usm2fsm1v2 | career_totals_regular_season | gp          |
+| ii54ianm3v79gn67r90uj9usm2fsm1v2 | season_totals_post_season    | gp          |
+| ii54ianm3v79gn67r90uj9usm2fsm1v2 | season_totals_regular_season | gp          |
++----------------------------------+------------------------------+-------------+
+```
+
+From these results, we can see there were five unique commits to this database that updated the `gp` statistic. If we
+wanted to narrow in on a specific table in this list, we could use the following query to see how many times each statistic
+was updated over the course of all our commits:
+
+```sql
+SELECT column_name, count(commit_hash) as total_column_changes 
+FROM dolt_column_diff 
+WHERE table_name = 'rankings_regular_season' 
+GROUP BY column_name;
+```
+
+```text
++-------------------+----------------------+
+| column_name       | total_column_changes |
++-------------------+----------------------+
+| rank_fg3_pct      | 6                    |
+| rank_eff          | 6                    |
+| season_id         | 6                    |
```

Comment:
(minor) this is still a good example, I'm just wondering if there's more of a punchline somewhere we could deliver... For example... if we used a query to determine which columns haven't ever been changed in the past <time period> or something it might be a little stronger. This isn't a bad example at all, I'm just wondering if there's some other example that people would read and say "oh wow! that's really useful!". I think we're missing a piece here connecting the data back to a reason why this is helpful/useful for customers. I know the example for docs is a little contrived, and it's not bad, just seems like maybe we can think a little more and see if there's something punchier we can come up with.